usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/port/GtfsDataRepository.java

```diff
@@ -27,4 +28,8 @@
     Route addRoute(final Route newRoute) throws IllegalArgumentException;
```

Comment:
Bad merge here I think, the order "add, get" will be re-established in the next commit